src/coreComponents/physicsSolvers/fluidFlow/wells/WellControls.cpp

```diff
@@ -53,7 +53,7 @@ WellControls::WellControls( string const & name, Group * const parent )
     setDescription( "Well control. Valid options:\n* " + EnumStrings< Control >::concat( "\n* " ) );
 
   registerWrapper( viewKeyStruct::targetBHPString(), &m_targetBHP ).
-    setDefaultValue( -1 ).
+    setDefaultValue( 0.0 ).
```

Comment:
Will this change the value of the `m_targetBHP <= 0.0` check? Same value for `-1` or `0.0`?

Reply:
This PR should not change the value of the `m_targetBHP <= 0.0`, and we don't want this to change for now.